### DISCUSSION TITLE
Installing OKD on ARM64 with FCOS as machine-os

### DISCUSSION THREAD

**Discussion by hisham-vaporio** [okd-project/okd#1915] [Q&A]
Hello all,

I am attempting to get OKD64 installed on an ARM64 baremetal. I was able to get it installed on an x86_64 baremetal. However since there are no arm64 target releases, I attempted using an OCP openshift-installer and it appears that it is expecting RHCOS and not FCOS, as it was not pivoting. (i think that's the issue, it doesn't successfully complete the pre-pivot service and both bootstrap and install-to-disk continue to wait for `done` or the existance of `oc`, `kubectl` and `hyperkube` )

Do I need to build OKD's openshift-installer and oc-tools to make it usable in my situation, or are there env overrides that I can use that'll help me?

Thank you for your time and help on this.

**Comment by codespearhead** [comment]
I suggest closing this discussion in favor of #1165 , and commenting your findings there.
